helm-chart/binderhub/schema.yaml

```diff
@@ -417,16 +417,20 @@ properties:
       storageDriver:
         type: string
         description: |
-          TODO
+          Docker storage driver
       resources: *resources-spec
-      hostSocketDir:
+      hostSocketDir: &hostSocketDir-spec
         type: string
         description: |
-          TODO
-      hostLibDir:
+          Host directory where the container socket will be located
+      hostLibDir: &hostStorageDir-spec
```

Comment:
We could change this to the more generic `hostStorageDir` at some point to match the pink parameter names, but that's a breaking change.